### PR TITLE
Add new compatibility-yaml with CSI 2.5.1 support

### PR DESCRIPTION
**Pull request by asifdxtreme**
**What type of PR is this?**
 /kind enhancement

**What this PR does / why we need it**: This PR adds the new compatibility-yaml with CSI 2.5.1 support, the v0.2.0 operator can be used with the new compatibility yaml to get CSI 2.5.1 support.

### CODE REVIEW

**Review by ridaz — APPROVED** [review]
LGTM, Thanks

**Review by lgayatri — APPROVED** [review]
Looks Good

**Comment by jvrahav** [comment]
have we assessed if we need changes related to custom kubelet path w.r.t 2.5.x

File: artifacts/compatibility-yaml/compatibility-v0.2.1.yaml [hunk 1]
```diff
@@ -0,0 +1,98 @@
+{
+  "CSI" : {
+    "2.5.1": {
```
Comment (jvrahav) [inline artifacts/compatibility-yaml/compatibility-v0.2.1.yaml:3]: have we assessed if vdo code needs any change w.r.t custom kubelet path for version 2.5.x

Comment (asifdxtreme) [reply artifacts/compatibility-yaml/compatibility-v0.2.1.yaml:3]: checked the latest version and validated the feature

**Pull request merged**